packages/dappmanager/src/modules/compose/editor.ts

```diff
@@ -89,6 +89,65 @@ class ComposeServiceEditor {
     }));
   }
 
+  removeNetworkAliases(
```

Comment:
You must unit test key sensitive logic like this that is very easy to test as a pure function